docs/formats.md

```diff
@@ -71,6 +71,8 @@ A special file configuration is `filter`, which will filter the tokens before th
 }
 ```
 
+The token/property that is passed to the filter function has already been [transformed](transforms.md) and has [default metadata](properties.md?id=default-property-metadata) added by Style Dictionary.
```

Comment:
not sure default is the right adjective. as opposed to optional metadata? might be worth considering the wording here.